apps/web/src/stores/documentVersions.ts

```diff
@@ -1,8 +1,11 @@
 'use client'
 
-import { DocumentVersion, HEAD_COMMIT } from '@latitude-data/core'
+/* import { HEAD_COMMIT, type DocumentType } from '@latitude-data/core' */
```

Comment:
I think this was other problem. Core package contains server code and when we do import things from it in client it fails. 

This is very surprising to be honest